doc/howtos/building-images.rst

```diff
@@ -230,17 +241,34 @@ The following instructions assume that swupd is not used.
 
 .. TODO: document how to create custom image recipes based on ostro-image.bbclass.
 
+.. _`ostro-image.bbclass`: https://github.com/ostroproject/meta-ostro/blob/master/meta-ostro/classes/ostro-image.bbclass
 
-Installing Additional Packages without swupd
---------------------------------------------
+Installing Additional Packages (without swupd)
```

Comment:
@gvancuts is right, OSTRO_IMAGE_EXTRA_FEATURES and OSTRO_IMAGE_EXTRA_INSTALL work for both kinds of images, because both derive from ostro-image.bbclass.

It starts to get different once variables specific to ostro-image-noswupd.bb or ostro-image-swupd.bb are involved.

The original motivation for focusing only on images "without swupd" was that swupd was still new and it was unclear how much was supposed to be written about it, and in which way.